cmd/fdo_server/server.go

```diff
@@ -628,7 +671,7 @@ func ownerModules(ctx context.Context, guid protocol.GUID, info string, chain []
 		if cmdDate && slices.Contains(modules, "fdo.command") {
 			if !yield("fdo.command", &fsim.RunCommand{
 				Command: "date",
-				Args:    []string{"--utc"},
+				Args:    []string{"+%s"},
```

Comment:
Any particular reason for this change?

Reply:
I don't know how that went in actually :confused:

Reply:
https://github.com/fido-device-onboard/go-fdo-server/pull/20